### PR TITLE
Explain that Clojure only uses prefix notation

### PR DESCRIPTION
**Pull request by danevans**
Addresses #148

### CODE REVIEW

File: slides/module1.html [hunk 1]
```diff
@@ -134,6 +134,10 @@ <h3 class="slide-title slide">Arithmetic</h3>
           <h3 class="slide-title slide">Infix vs. prefix
           notation</h3>
 
+          <p>Languages such as <strong>JavaScript</strong> use <strong>infix</strong>
+            notation<br />
```
Comment (davidchambers) [inline slides/module1.html]: Why the `<br />`?

**Comment by davidchambers** [comment]
The content looks good to me, but I'm interested in the choice of markup. Why so `<strong>`? :)

**Comment by danevans** [comment]
screen shot 2015-04-21 at 12 31 08 pm
I threw in the `<br />` because it looked weird to me to have "while" as the last word on the line but I could remove it. 
The `<strong>`s just emphasized the important words for me. I wanted to emphasize those words to help connect them to the example code blocks.

**Comment by kpfell** [comment]
Yeah, I'd rather the `<strong>` stayed; it makes the text a bit easier to parse. Add a comma after "notation", though?

Otherwise, :+1:

**Comment by davidchambers** [comment]
The screenshot helped thanks, @danevans! Looks good.

> Add a comma after "notation", though?

Yes, I agree.

**Comment by danevans** [comment]
I modified the change to add the comma.

**Comment by kpfell** [comment]
I am interpreting @davidchambers's "Looks good" as a second :+1:. :wink: 

Merged!

**Comment by davidchambers** [comment]
Yep! :+1:

**Comment by yokolet** [comment]
This sentence was once dropped when I changed the slides rendering way. At this occasion, I made this back, http://clojurebridge.github.io/curriculum/outline/simple_values.html#/9 .

**Pull request merged**